core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -1,7 +1,9 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak">
   <li ng-if="showPublishButton()">
     <div class="btn-group" style="margin-right: 10px; margin-top: 8px;"
-         title="<[getPublishExplorationButtonTooltip()]>">
+         uib-tooltip="<[getPublishExplorationButtonTooltip()]>"
+         tooltip-placement="bottom"
+         ng-class="{'div-disable': isExplorationLockedForEditing() || countWarnings()}" ng-click="showTooltip()">
```

Comment:
Just a query. If the button is disabled, will the ng-click event get triggered if clicked? If not when is the showTooltip function triggered? And why do we need it?

Reply:
The `ng-click` on the div element gets triggered since there is no disable on the div.

Reply:
Ah I see. Thanks for the explanation. But on hover itself the tooltip is shown right? how does the click help?

Reply:
In the current working, when the user clicks the tooltip is gone but in current scenario, the tooltip stays on click. I am not sure if this is needed since the button is disabled which already implies user cannot click on it.

Reply:
I think the tooltip should show immediately on hover as it does. I think disappearing on click is ok. (I tried on google docs, and their tooltips disappear on click). Also, when disabled, anyway the clicks would not register I think. 

After removing, if on clicking the disabled button, the tooltip disappears, then lets keep this functionality. If on clicking the disabled button, the tooltip stays, let's get rid of the ng-clicks.

Reply:
If that is the case, we can just use the title attribute and that will work fine. I will update the PR accordingly!

Reply:
Done, PTAL!